sdk/core/Azure.Core/src/DynamicData/MutableJsonDocument.cs

```diff
@@ -110,32 +120,34 @@ private static void Write(Stream stream, ReadOnlySpan<byte> buffer)
         public static MutableJsonDocument Parse(ReadOnlyMemory<byte> utf8Json)
         {
             var doc = JsonDocument.Parse(utf8Json);
-            return new MutableJsonDocument(doc, utf8Json);
+            return new MutableJsonDocument(doc, utf8Json, default);
```

Comment:
I don't have the full context of how this method is used, but would it make sense to also accept the options on the Parse method?

Reply:
We can ... since it's an internal type, I didn't want to spend too much time on it at this point, but happy to update.

Reply:
Well, this sort of led me to my other thought - given this is all internal stuff, what are we gaining from these changes? Is it just to prepare for exposing this option in the future?

Reply:
Interesting.. okay thanks for looking into that.